snapshot/manager.go

```diff
@@ -31,6 +31,13 @@ const (
 	loadSnapshotsConcurrency = 50 // number of snapshots to load in parallel
 )
 
+var unavailableKeyMap = map[string]struct{}{
```

Comment:
instead of restricting, can we have label keys prefixed with `tag:` so all key/value pairs are effectively allowed?

```
{
   "type": "snapshot",
   "username": "alice",
   "hostname": "wonderland",
   "path": "/rabbit/hole",
   "tag:foo": "bar",
   "tag:baz": "qux",
}
```

Reply:
@jkowalski That also works. I had initially suggested the validation to prevent clobbering _"reserved"_ labels.